packages/tocco-test-util/src/testSetup/bundle.js

```diff
@@ -0,0 +1,9 @@
+import { configure } from 'enzyme'
```

Comment:
Is there no linting rule for the spaces inside `{...}` or is this file excluded from the linting? (as far as I can tell, we prefer the style without spaces)